.gitignore

```diff
@@ -10,6 +10,7 @@ src/server/node_modules/**/*
 src/client/node_modules/**/*
 npm-debug.*
 web-build/
+.env
```

Comment:
Will send .env file in Slack

Reply:
just commit?

Reply:
very bad practice to commit .env files, github will report vulnerabilities like crazy if I commit it